pkg/agent/svid/rotator_test.go

```diff
@@ -106,7 +106,7 @@ func (s *RotatorTestSuite) TestRun() {
 	s.Assert().Equal(key, state.Key)
 
 	cancel()
-	s.Require().NoError(t.Wait())
+	s.Require().Equal(context.Canceled, t.Wait())
```

Comment:
I assume this change in behavior has no other impact?

Reply:
It shouldn't.